www/src/utils/createOgUrl.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import {urlRoot} from 'site-constants';
```

Comment:
What is `site-constants` ? Shouldn't this just be `constants` (to refer to the `constants.js` file in the root directory)?

Reply:
Disregard this comment. I see the note about this.